tools/project_manager/add_files.php

```diff
@@ -103,20 +103,10 @@
     $source_project_dir = $abs_source;
 }
 
-// Attempt to make everything all-readable.
```

Comment:
I don't know why we do this. The script needs read permissions from the source files but if the `www-data` webserver user created the files (which would be the case from an extracted zip or something uploaded through Remote File Manager) then it has read permissions on it already.

This might be an artifact from when we allowed ftp uploads but that is a long time gone.

Reply:
To change the permissions on the files we have to have write access, and if we have write access we probably have read access. And the only thing we actually need is read access to copy it from this location to the new location and when its copied it will be owned by the `www-data` user, so I don't think this is needed at all.